### ISSUE TITLE
Labelrank of populated places is the labelrank of the admin-0, not the populated place itself

### ISSUE THREAD

**Issue by jkunimune** [nvkelso/natural-earth-vector#872]
In all of the populated places datasets, the labelrank attribute, like the scalerank attribute, is supposed to represent the relative importance of the record according to its population, its administrative status, and other things (see [this forum post](https://www.naturalearthdata.com/forums/topic/label-ranks/)). However, in reality, it appears to be the labelrank of the admin-0 that contains it.  So all research stations in Antarctica are labelrank 0, all cities in the US are labelrank 1 (regardless of size), and so on.  This is not a huge problem since labelrank is arguably redundant with scalerank for populated places, but it's also clearly not intentional, and somewhat confusing.  For illustration, here are scatter plots of the records in 50m populated places, with rank indicated by color and size.

![labelrank](https://github.com/nvkelso/natural-earth-vector/assets/5607801/0afec3ba-d448-4094-8cf7-e63c9775ac21)
![scalerank](https://github.com/nvkelso/natural-earth-vector/assets/5607801/896464b1-0abb-44e9-9bf4-47042e613d27)

**Comment by nvkelso** [comment]
Agree. Labelrank hasn't been maintained in a very long time, isn't useful, and should be removed next major version.

Use scalerank or min_zoom instead.
